docker-test-framework/docker-compose.yml

```diff
@@ -26,18 +12,6 @@ services:
         - "8080:8080" 
       networks:
         - 4-2_irodsnet
```

Comment:
Please open an issue to rename this network.

Reply:
It looks like the newest jargon has the network as simply "irodsnet".  I'll update it.

The README.md in this directory needs updating too as DOCKERTEST.md is now in docker-test-framework of jargon.